src/main/java/core/world/WorldEntityWithSprite.java

```diff
@@ -0,0 +1,30 @@
+package core.world;
+
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.physics.box2d.BodyDef;
+import com.badlogic.gdx.physics.box2d.FixtureDef;
+
+public class WorldEntityWithSprite extends WorldEntity {
```

Comment:
Don't know if this is necessary as keeping it within the WorldEntity doesn't add too much bulk

Reply:
many WorldEntity objects need no sprite